### PR TITLE
Actions/deploy storybook

### PR DESCRIPTION
**Pull request by dprokop**
- [x] Deploy to GCP for each PR
- [x] Put deployment into a "canary" folder when merging into master
- [x] Create version folders on releases

### CODE REVIEW

**Comment by tskarhed** [comment]
Release and merge workflows still needs to be tested, does anyone have an idea on how to do that?

**Review by mckn — COMMENTED** [review]
I think it looks good! Great job! 

Regarding testing, just release the kraken:
ship-it

**Comment by mckn** [comment]
> Release and merge workflows still needs to be tested, does anyone have an idea on how to do that?

Also - if you want to test those flows. You could just change the trigger. So they get triggered by another action that you easier can control. That should be enough of testing to make sure that the flow is working.

**Comment by tskarhed** [comment]
> > Release and merge workflows still needs to be tested, does anyone have an idea on how to do that?
> 
> Also - if you want to test those flows. You could just change the trigger. So they get triggered by another action that you easier can control. That should be enough of testing to make sure that the flow is working.

Sounds like a good plan! The release setup will probably be done in CircleCI, I am aiming for merge on Monday in case there is any trouble.

**Review by aknuds1 — APPROVED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
     "@babel/preset-env": "7.8.4",
     "@babel/preset-react": "7.8.3",
     "@babel/preset-typescript": "7.8.3",
+    "@actions/core": "^1.1.1",
+    "@actions/github": "^1.1.0",
```
Comment (dprokop) [inline package.json]:
```suggestion
    "@actions/core": "1.1.1",
    "@actions/github": "1.1.0",
```

File: .github/workflows/pr-deploy-storybook.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Build Grafana Storybook
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled]
```
Comment (dprokop) [inline .github/workflows/pr-deploy-storybook.yml]: I think removing `labeled` here might be nice

**Pull request closed without merging**